examples/test_error.py

```diff
@@ -101,13 +101,17 @@
     # }}}
 
     # {{{Normalization
-    frq_slice = integrate_limits(select_pathway(s, signal_pathway))
+    frq_slice = integrate_limits(select_pathway(s, signal_pathway),
+            convolve_method='Gaussian')
     s_integral = s["t2":frq_slice].C # the "official" copy of the integral
     s_integral = select_pathway(s_integral, signal_pathway)
     s_integral.integrate("t2")
     avg_d = s_integral.C.mean().item()
     s_integral /= avg_d
+    # the line below changes the frequency slice for some reason
     s /= avg_d
+    frq_slice = integrate_limits(select_pathway(s, signal_pathway),
+            convolve_method='Gaussian')
```

Comment:
after dividing by `avg_d` for some reason the `frq_slice` changes... in lines 104-105, it is returned as `-43.73959504  55.45762978` but after line 112 it is returned as `-43.73959504  75.29707474` - not sure why a normalization would have that effect, but by setting `frq_slice` after the division, this matches the `frq_slice_check` returned by `integral_w_errors` and satisfies the assert condition